inline_scan.sh

```diff
@@ -535,6 +535,14 @@ save_and_copy_images() {
 
     docker save "${SCAN_IMAGES[0]}" -o "${save_file_path}"
 
+    echo "docker save is in progress..."
+    while [[ ! -s "${save_file_path}" ]]; do
```

Comment:
@haresh-suresh  I suggest re-opening this PR. In case of a failure in docker save ,this will spin indefinitely. We need number of retries or a timeout here.

cc @yathi-sysdig , @ProvoK

Reply:
I will put out another quick PR 👍

Reply:
We tried this with a image > 5.5GB (inline v0.5.1). And the `docker save` took around 10-15mins. I do agree we need a retry count to not spin indefinitely but if we exhaust all retries because its a huge image, then failing the script might also. not be a good experience. @omer-sd what do you think?

The long term fix should be to use the modified slim version of `node-analyzer` and get rid of `docker save` which is taking a lot of time.

Reply:
@haresh-suresh, an alternative approach might be to send standard output to the file instead of `-o`:
```sh
docker save "${SCAN_IMAGES[0]}" > ${save_file_path}
```

Reply:
Most of the times, I'd say the CI/CD plugins like Jenkins would have a timeout like 1hr or 30 mins and should take care of this.

Reply:
@yathi-sysdig This is a good point, in fact I forgot that in the past retries have gotten us in trouble too. Since we can't guarantee execution time.

Reply:
@haresh-suresh, I'll expand on my comment. While `docker save` might spawn, allowing the command to return before finishing the write to file, the `>` of stdout to a file is not asynchronous.

Reply:
Maybe what we should do is have more distinct error codes for stuff like retries / timeout. 
And then just pipe something out like "Timeout exceeded for docker save" ?